src/test/java/com/microsoft/sqlserver/testframework/DBConnection.java

```diff
@@ -59,6 +78,29 @@ void getConnection(String connectionString) {
         }
     }
 
+    /**
+     * establish connection
+     * 
+     * @param connectionString
```

Comment:
`@param dataSource`

Reply:
I'll take care of these changes in next PR 👍